### ISSUE TITLE
Signatures not loading or not working

### ISSUE THREAD

**Issue by mphasize** [phpMussel/phpMussel#178]
### Server information. 
php.7.2-apache docker image with various extensions installed.

### phpMussel version used (if known, please also include the exact commit hash).
1.7.0 installed via Composer

### Signatures version (if known).
Freshly generated using sigtool and latest from Signatures repository

---
## Issue
`pdf_standard_testfile.pdf` doesn't trigger a problem when scanned.

### Expected behavior
If I understood it correctly, the test file should trigger a problem report.

### Description

Hello, I stumbled across phpMussel today and wanted to give it a try, as it looks like a very neat solution for virus scanning. After the installation via composer and then setting up the config, I tried a few different ways to trigger an upload block or problem report, but I can't get the test files to fail on the actual signatures.

I managed to trigger various chameleon problems with mismatched file extensions, but the ascii test file and the pdf test file don't trigger. My config.ini `Active` setting looks like `Active='clamav.db,clamav_pdf.db'` and I tried with the signature files downloaded from the Signatures repository as well as with the self-generated ones using Sigtool.

What I am not 100% sure of, is whether the Active signature files are actually loaded and checked. I tried various ways to specify the `Active` setting and clearly mislabelled signature files didn't produce an error. Is there any way to debug which signature files the uploaded file is checked against?

Any other ideas, why the test file would not trigger?
Any help is highly appreciated.

**Comment by DanielRuf** [comment]
Hi @mphasize,

I think there are issues about the same problem.
For example https://github.com/phpMussel/phpMussel/issues/172

We also have a scan log.

**Comment by mphasize** [comment]
@DanielRuf thanks for pointing it out. #172 was apparently solved by downgrading to PHP 7.0 and I can see no further ideas as to what the issue could be in the thread. 

My scan log for the file in question shows:
```
Fri, 28 Dec 2018 21:35:22 +0100 Started.
> Checking '/srv/app/app/..//uploads/tmp/pdf_standard_testfile.pdf' (FN: d8c6fd32; FD: 8c05e37f):
-> No problems found.
Fri, 28 Dec 2018 21:35:22 +0100 Finished.

```

**Comment by mphasize** [comment]
Ok, looks like I found the problem thanks to this https://github.com/phpMussel/phpMussel/issues/148 explanation.

In order to get the test file to trigger, I also had to add the `phpmussel_pdf.db` signature file to the `Active` setting. I had tried this before with negative results, when I didn't realise that internal caching prevented a rescan with the new signature configuration.

So after adding the phpmussel signature file AND deleting the cache (or setting `scan_cache_expiry=0`) it seems to work now.
